src/app/router/Welcome/Welcome.tsx

```diff
@@ -31,16 +33,15 @@ function getRoutes(
     {
       path: "/choose-path",
       name: i18n.t("translation:welcome.nav.connect"),
+      element: passwordExists ? (
```

Comment:
I think this change stopped the Alby login / create and choose connector links no longer work